src/Components/Subscriptions/index.jsx

```diff
@@ -71,6 +86,16 @@ const Subscriptions = () => {
   //     throw new Error(error);
   //   }
   // };
```

Comment:
The commented function i left last week, now i used it in the FORM, you can delete it